src/java/org/apache/cassandra/service/accord/AccordCommandStore.java

```diff
@@ -105,6 +191,9 @@ public boolean inStore()
     @Override
     protected void registerHistoricalTransactions(Deps deps)
     {
+        // TODO (impl) : Its not clear why this exists
+        // accord.coordinate.CoordinateSyncPoint.coordinate(accord.local.Node, accord.primitives.TxnId, accord.primitives.Seekables<?,?>)
```

Comment:
I ended up implementing this... `CommandsForRanges` now has a `local` and `remote` tracking case for this... for `remote` we "learn" the ranges by merging, but for `local` we trust the local txn register path.

I think I have the logic mostly matching InMemmoryCommandStore, but did refactor to keep things simple... for one the logical `mapReduceForRange` will include the history in all cases, but since I register as `NotWitnessed` this causes `minStatus == null && test_deps == ANY_DEP` to have basically the same semantics...  this allows a cleaner setup where I don't need to have 2 different code paths